src/frontend/mod.rs

```diff
@@ -904,7 +906,11 @@ pub mod tests {
             gov_id.get("_signer").unwrap(),
             pay_stub.get("_signer").unwrap(),
         )?;
-        assert!(request.exact_match_pod(&*kyc.pod));
+        assert!(request.exact_match_pod(&*kyc.pod).is_ok());
```

Comment:
This pattern is executing exact_match_pod twice.  I think the unwrap() below is sufficient to cover this assert.
Same pattern repeats below.